Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerMentionCellView_Tests.swift

```diff
@@ -0,0 +1,234 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerMentionCellView_Tests: XCTestCase {
+    /// Default reference width for the cell.
+    private static var defaultCellWidth: CGFloat = 374
```

Comment:
nit: You know this could be any artificial number? :D Why 374? Why not 300? 400? 450? 😅

Reply:
It's the same value without needing of re-recording snapshots :trollface: Ended up re-recording snapshot anyways but this kept around :D

Reply:
Please change into a better number. This looks oddly specific and raises questions.